server/src/test/java/org/opensearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -794,6 +794,108 @@ public void testConcurrentSnapshotCreateAndDeleteOther() {
         }
     }
 
+    public void testTransportGetSnapshotsAction() {
+        setupTestCluster(randomFrom(1, 3, 5), randomIntBetween(2, 10));
+
+        String repoName = "repo";
+        final String[] snapshotsList = { "snapshot-1", "snapshot-2" };
+        final String[] indexList = { "index-1", "index-2" };
+        final boolean[] snapshotRequestMode = { true, false };
+        final int shards = randomIntBetween(1, 10);
+
+        TestClusterNodes.TestClusterNode clusterManagerNode = testClusterNodes.currentClusterManager(
+            testClusterNodes.nodes.values().iterator().next().clusterService.state()
+        );
+
+        for (int i = 0; i < snapshotsList.length; i++) {
+            final StepListener<CreateSnapshotResponse> createSnapshotResponseStepListener = new StepListener<>();
+            final String snapshot = snapshotsList[i];
+            final String index = indexList[i];
+            continueOrDie(
+                createRepoAndIndex(repoName, index, shards),
+                createSnapshotResponse -> client().admin()
+                    .cluster()
+                    .prepareCreateSnapshot(repoName, snapshot)
+                    .setWaitForCompletion(true)
+                    .execute(createSnapshotResponseStepListener)
+            );
+        }
+        deterministicTaskQueue.runAllRunnableTasks();
+
+        TransportAction getSnapshotsAction = clusterManagerNode.actions.get(GetSnapshotsAction.INSTANCE);
+        TransportGetSnapshotsAction transportGetSnapshotsAction = (TransportGetSnapshotsAction) getSnapshotsAction;
+        for (boolean mode : snapshotRequestMode) {
+            GetSnapshotsRequest repoSnapshotRequest = new GetSnapshotsRequest().repository(repoName)
+                .snapshots(snapshotsList)
+                .ignoreUnavailable(mode)
+                .verbose(mode);
+
+            transportGetSnapshotsAction.execute(null, repoSnapshotRequest, ActionListener.wrap(repoSnapshotResponse -> {
+                assertNotNull("Snapshot list should be registered", repoSnapshotResponse.getSnapshots());
```

Comment:
change the message to "Snapshot list should be non null". Also, change in below test case

Reply:
Ack.